### ISSUE TITLE
[bbc] world-europe-60630352: Unable to extract playlist data

### ISSUE THREAD

**Issue by RalphCorderoy** [yt-dlp/yt-dlp#2964]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.02.04**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

UK

### Description

`bbc` says to report the issue here. `--write-pages` file attached.

### Verbose log

```shell
$ ./yt-dlp -vU --write-pages -F https://www.bbc.co.uk/news/world-europe-60630352
[debug] Command-line config: ['-vU', '--write-pages', '-F', 'https://www.bbc.co.uk/news/world-europe-60630352']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, err UTF-8, pref UTF-8
[debug] yt-dlp version 2022.02.04 [c1653e9ef] (zip)
[debug] Python version 3.6.3 (CPython 64bit) - Linux-4.13.0-46-generic-x86_64-with-Ubuntu-17.10-artful
[debug] exe versions: ffmpeg 3.3.4-2, ffprobe 3.3.4-2, rtmpdump 2.4
[debug] Optional libraries: sqlite
[debug] Proxy map: {}
Latest version: 2022.02.04, Current version: 2022.02.04
yt-dlp is up to date (2022.02.04)
[debug] [bbc] Extracting URL: https://www.bbc.co.uk/news/world-europe-60630352
[bbc] world-europe-60630352: Downloading webpage
[bbc] Saving request to world-europe-60630352_https_-_www.bbc.co.uk_news_av_world-europe-60630352.dump
ERROR: [bbc] world-europe-60630352: Unable to extract playlist data; please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U; please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U
  File "./yt-dlp/yt_dlp/extractor/common.py", line 612, in extract
    ie_result = self._real_extract(url)
  File "./yt-dlp/yt_dlp/extractor/bbc.py", line 1264, in _real_extract
    webpage, 'playlist data'),
  File "./yt-dlp/yt_dlp/extractor/common.py", line 1186, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
```

**Comment by RalphCorderoy** [comment]
[world-europe-60630352_https_-_www.bbc.co.uk_news_av_world-europe-60630352.dump.txt](https://github.com/yt-dlp/yt-dlp/files/8191139/world-europe-60630352_https_-_www.bbc.co.uk_news_av_world-europe-60630352.dump.txt)

**Comment by pukkandan** [comment]
Fixed by 50e93e03a7ca6ae35a319ea310104f7d6d91eee3
